src/kubernetes.cc

```diff
@@ -78,8 +78,14 @@ class KubernetesReader::NonRetriableError
 
 KubernetesReader::KubernetesReader(const Configuration& config,
                                    HealthChecker* health_checker)
+    : KubernetesReader(config, health_checker, SleeperImpl::New()) {}
+
+KubernetesReader::KubernetesReader(const Configuration& config,
+                                   HealthChecker* health_checker,
+                                   std::unique_ptr<Sleeper> sleeper)
```

Comment:
Can we rename `sleeper` to be a bit more descriptive? This only seems to be used during retry backoffs.

Reply:
Happy to rename. What would you like?